### PR TITLE
Release test

### PR DESCRIPTION
**Pull request by Barnes2197**

### CODE REVIEW

File: src/controllers/announcement.js [hunk 1]
```diff
@@ -3,40 +3,148 @@ import Announcement from "../models/announcement";
...
+    httpResponse.failureResponse(res, e)
   }
-  if(challenge)
+}
+
+const read = async (req, res) =>
+{
+  try
+  {
+    const title = req.body.title
```
Comment (Tommy2016x) [inline src/controllers/announcement.js]: destructure this

File: src/controllers/announcement.js [hunk 2]
```diff
@@ -3,40 +3,148 @@ import Announcement from "../models/announcement";
...
+    httpResponse.failureResponse(res, e)
   }
+
+}
+
+const announce = async (req, res) =>
+{
+  try
+  {
+    const title = req.body.title
```
Comment (Tommy2016x) [inline src/controllers/announcement.js]: desctructure

File: src/controllers/announcement.js [hunk 3]
```diff
@@ -3,40 +3,148 @@ import Announcement from "../models/announcement";
...
+    console.log(e)
+    httpResponse.failureResponse(res, e)
+  }
+}
+
+const remove = async (req, res) =>
+{
+  try
+  {
+    const title = req.body.title
```
Comment (Tommy2016x) [inline src/controllers/announcement.js]: desctructure

File: src/routes/index.js [hunk 4]
```diff
@@ -88,6 +87,9 @@ apiRouter.get("/workshop/read", workshop.read);
 /* ------ Live-Site Announcements ------ */
  apiRouter.post("/announcement", announcement.create);
  apiRouter.get("/announcement", announcement.read);
+ apiRouter.put("/announcement/update", announcement.update);
```
Comment (Tommy2016x) [inline src/routes/index.js]: How are these going to be called? Through the shell admin? or another client

File: src/routes/index.js [hunk 5]
```diff
@@ -88,6 +87,9 @@ apiRouter.get("/workshop/read", workshop.read);
...
+ apiRouter.delete("/announcement/remove", announcement.remove);
+ apiRouter.post("/announcement/announce", announcement.announce);
```
Comment (Tommy2016x) [inline src/routes/index.js]: add adminAuth middleware to all announcement routes instead of the get

Comment (Barnes2197) [reply src/routes/index.js]: added to all routes except the git route

File: src/controllers/application.js [hunk 6]
```diff
@@ -509,44 +509,6 @@ const resetPassword = async (req,res) => {
     }
 }
 
-const confirmEmail = async (req,res) => {
```
Comment (Tommy2016x) [inline src/controllers/application.js]: Is removing this function intentional?

Comment (Barnes2197) [reply src/controllers/application.js]: No it isn’t actually idk how that happened

Comment (Tommy2016x) [reply src/controllers/application.js]: add it back then and make sure its intended functionality is still in place

Comment (Barnes2197) [reply src/controllers/application.js]: Ok so i did delete it and rewrote it a different way

**Pull request merged**